hospital.java

```diff
@@ -0,0 +1,14 @@
+
+public abstract class hospital {

```

Comment:
You could use an Interface here because you only define method signatures. Usually we use abstract classes when we want to share implemented methods or data between the child classes.